helm/values-prod.yaml

```diff
@@ -0,0 +1,163 @@
+    tag: "0.0.0"
+    pullPolicy: Always
+  livenessProbe:
+    httpGet:
+      path: /actuator/health/liveness
+      port: 8080
+    initialDelaySeconds: 40
+    failureThreshold: 6
+    periodSeconds: 10
+  readinessProbe:
+    httpGet:
+      path: /actuator/health/readiness
+      port: 8080
+    initialDelaySeconds: 40
+    failureThreshold: 6
+    periodSeconds: 10
+  deployment:
+    create: true
+    replicas: 1
+  service:
+    create: true
+    type: ClusterIP
+    ports:
+      - 8080
+  ingress:
+    create: true
+    host: "itnprod.pay-wallet.internal.platform.pagopa.it"
+    path: /pagopa-payment-wallet-cdc-service/(.*)
+    servicePort: 8080
+  serviceAccount:
+    create: false
+    annotations: { }
+    name: ""
+  podAnnotations: { }
+  podSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  securityContext:
+    allowPrivilegeEscalation: false
+  resources:
+    requests:
+      memory: "512Mi"
+      cpu: "300m"
+    limits:
+      memory: "512Mi"
+      cpu: "600m"
+  autoscaling:
+    enable: false
+    minReplica: 1
+    maxReplica: 1
+    pollingInterval: 10 # seconds
+    cooldownPeriod: 50 # seconds
+    triggers:
+      - type: cpu
+        metadata:
+          # Required
+          type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
+          value: "75"
+  terminationGracePeriodSeconds: 30
+  strategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxSurge: "100%"
+      maxUnavailable: "100%"
+  envConfig:
+    OTEL_SERVICE_NAME: "pagopa-payment-wallet-cdc-service"
+    OTEL_SERVICE_ENVIRONMENT: "prod"
+    OTEL_RESOURCE_ATTRIBUTES: "deployment.environment=prod"
+    OTEL_EXPORTER_OTLP_ENDPOINT: "https://weuprod.kibana.internal.platform.pagopa.it/apm"
+    OTEL_EXPORTER_OTLP_PROTOCOL: http/protobuf
+    OTEL_TRACES_EXPORTER: otlp
+    OTEL_METRICS_EXPORTER: otlp
+    OTEL_LOGS_EXPORTER: otlp
+    OTEL_TRACES_SAMPLER: "always_on"
+    CDC_QUEUE_NAME: pagopa-p-itn-pay-wallet-cdc-queue
+    CDC_QUEUE_TTL_SECONDS: "604800" #7 days
+    CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS: "0"
+    MONGO_HOST: pagopa-p-itn-pay-wallet-cosmos-account.mongo.cosmos.azure.com
+    MONGO_USERNAME: pagopa-p-itn-pay-wallet-cosmos-account
+    MONGO_PORT: "10255"
+    MONGO_SSL_ENABLED: "true"
+    MONGO_DB_NAME: "payment-wallet"
+    MONGO_MIN_POOL_SIZE: "0"
+    MONGO_MAX_POOL_SIZE: "50"
+    MONGO_MAX_IDLE_TIMEOUT_MS: "600000"
+    MONGO_CONNECTION_TIMEOUT_MS: "2000"
+    MONGO_SOCKET_TIMEOUT_MS: "10000"
+    MONGO_SERVER_SELECTION_TIMEOUT_MS: "2000"
+    MONGO_WAITING_QUEUE_MS: "2000"
+    MONGO_HEARTBEAT_FREQUENCY_MS: "5000"
+    CDC_LOG_EVENTS_COLLECTION_NAME: "payment-wallets-log-events"
+    CDC_LOG_EVENTS_OPERATION_TYPE: "insert,update,replace"
+    CDC_LOG_EVENTS_PROJECT: "fullDocument"
+    CDC_SEND_RETRY_MAX_ATTEMPTS: "1"
+    CDC_SEND_RETRY_INTERVAL_IN_MS: "1000"
+    CDC_STREAM_RETRY_MAX_ATTEMPTS: "15"
+    CDC_STREAM_RETRY_INTERVAL_IN_MS: "60000"
```

Comment:
1 minute retry interval seem fine to me here, is not a challenging value for the system considering that the CosmosDB is in premium tier.
Maybe it can be lowered but for the moment I would like to leave this parameter as is and tuning it after production deployment.
The step 1 release will be the production testing for all those features where no communication will be performed to data lake so we can use this deployment slot to test our configurations and tune it as needed

Reply:
A minute interval is fine for me, I meant to increase the max attempts to cover a (unlikely) connection problem that takes longer than 15 min